rules/scalac/private.bzl

```diff
@@ -0,0 +1,191 @@
+load("@rules_scala_annex//rules:providers.bzl", "ScalaConfiguration")
+load("//rules/common:private/utils.bzl", "strip_margin")
+load("//rules/common:private/utils.bzl", "write_launcher")
+load("//rules/scala:private/import.bzl", "create_intellij_info")
+
+_common_private_attributes = {
+    "_host_javabase": attr.label(
+        default = Label("@bazel_tools//tools/jdk:current_java_runtime"),
+        cfg = "host",
+    ),
+    "_java": attr.label(
+        default = Label("@bazel_tools//tools/jdk:java"),
+        executable = True,
+        cfg = "host",
+    ),
+    "_jar": attr.label(
+        default = Label("@bazel_tools//tools/jdk:jar"),
+        executable = True,
+        cfg = "host",
+    ),
+    "_jar_creator": attr.label(
+        default = Label("@rules_scala_annex//rules/common/third_party/bazel/src/java_tools/buildjar/java/com/google/devtools/build/buildjar/jarhelper:jarcreator_bin"),
```

Comment:
FYI, explicit Label() is not required here. I'm not sure what's most idiomatic.

Reply:
Yep. I'm not sure either. We should ping the Bazel team and then make sure we use whatever is most idiomatic.